node-definitions/arista/veos/vEOS.yaml

```diff
@@ -0,0 +1,64 @@
+id: vEOS
+general:
+  description: Arista vEOS
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - eth0
+      - eth2
+      - eth3
+      - eth4
+      - eth5
+      - eth6
+      - eth7
+      - eth9
+      - eth10
+      - eth11
+      - eth12
+      - eth13
+    serial_ports: 1
+    default_count: 13
+    loopback:
+      - l0
+ui:
+  description: |-
+    Arista vEOS
+
+    2 GB DRAM, 2 vCPU
+
+    [Link](https://www.arista.com/en/cg-veos-router/veos-router-overview)
+  visible: true
+  label_prefix: veos-
+  icon: switch
+  label: vEOS
+sim:
+  linux_native:
+    libvirt_domain_driver: kvm
+    driver: server
+    disk_driver: sata
+    ram: 2048
+    cpus: 2
+    cpu_limit: 100
+    nic_driver: virtio
+boot:
+  timeout: 1
```

Comment:
Wow, that's a fast boot time.  Does it really boot in one second?

Reply:
nope, this was canned I believe. I think the boot time is roughly 2-3 minutes.

Reply:
nope, generally 2-3 minutes.